packages/editor-ui/src/views/SetupWorkflowFromTemplateView/setupTemplate.store.ts

```diff
@@ -232,6 +232,20 @@ export const useSetupTemplateStore = defineStore('setupTemplate', () => {
 		templateId.value = id;
 	};
 
+	/**
+	 * Selects initial credentials for the template. Credentials
+	 * need to be loaded before this.
+	 */
+	const setInitialCredentialSelection = () => {
+		for (const credUsage of credentialUsages.value) {
+			const availableCreds = credentialsStore.getCredentialsByType(credUsage.credentialType);
+
+			if (availableCreds.length === 1) {
```

Comment:
should we avoid doing this for generic http credentials? I think we have some logic around that for automatically setting creds in NDV

Reply:
I couldn't find any sophisticated logic how those would be ignore. I added a simple exclude list for those credential types. Does that work @mutdmour ?

Reply:
Yup.. Looks good to me.. Sorry I don't recall specifics of that, but I find it really annoying usually when it auto selects these cred types.